src/reporting/clients/AggregationsClient.ts

```diff
@@ -0,0 +1,350 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import { RequiredError } from "../interfaces/Errors";
+import type { EntityListIterator } from "../../common/iterators/EntityListIterator";
+import { EntityListIteratorImpl } from "../../common/iterators/EntityListIteratorImpl";
+import { Collection, getEntityCollectionPage } from "../../common/iterators/IteratorUtil";
+import { OperationsBase } from "../../common/OperationsBase";
+import { AggregationProperty, AggregationPropertyCollection, AggregationPropertyType, AggregationPropertySingle, AggregationPropertyUpdate, AggregationPropertyCreate,
+         AggregationTableSet, AggregationTableSetCollection, AggregationTableSetSingle, AggregationTableSetUpdate, AggregationTableSetCreate,
+         AggregationTable, AggregationTableCollection, AggregationTableSingle, AggregationTableUpdate, AggregationTableCreate
+        } from "../interfaces/AggregationProperties";
+import type { IAggregationsClient } from "./IAggregationsClient";
+
+export class AggregationsClient extends OperationsBase implements IAggregationsClient {
+
+    public async getAggregationProperties (
+        accessToken: AccessToken,
```

Comment:
There are a lot of lint errors. Run `npm run lint` and fix all the errors that appear. The PR build validation will not pass with linter errors.